src/main/java/CalculatorTesting/TestTypes/InfixToPostfixCases/DoubleBrackets.java

```diff
@@ -0,0 +1,46 @@
+package CalculatorTesting.TestTypes.InfixToPostfixCases;
+
+import ShuntingYardCalculator.Calculator.CalculationSteps.InfixToPostfix;
+import ShuntingYardCalculator.Enums.Type;
+import javafx.util.Pair;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+import static CalculatorTesting.ParsePostfixString.parse;
+import static ShuntingYardCalculator.Calculator.EquationParser.parseEquationString;
+import static org.junit.Assert.assertEquals;
+
+public class DoubleBrackets {
+    private static ArrayList<Pair<String, Type>> DOUBLE_BRACKETS_INPUT;
+    private static ArrayList<Pair<String, Type>> DOUBLE_BRACKETS_OUTPUT;
+
+    @Test
+    public void test() {
```

Comment:
why do you have every test with test & setup function? why not use a base class?